Editor/SceneView/ChiselDefaultPlacementTools.cs

```diff
@@ -45,14 +45,16 @@ public override void OnSceneGUI(SceneView sceneView, Rect dragArea)
                 case ShapeExtrusionState.Modified:
                 case ShapeExtrusionState.Create:
                 {
+                    ChiselModelComponent model = generatedComponent ?
+                        generatedComponent.GetComponentInParent<ChiselModelComponent>() : null;
```

Comment:
I'm not sure what's going on here, is this a fix for something? Why was it changed? What didn't work?